mapbox/libjava-geojson/src/main/java/com/mapbox/services/commons/geojson/custom/PositionSerializer.java

```diff
@@ -37,9 +38,10 @@ public JsonElement serialize(Position src, Type typeOfSrc, JsonSerializationCont
     JsonArray rawCoordinates = new JsonArray();
 
     BigDecimal lat = new BigDecimal(src.getLatitude());
-    lat = lat.round(new MathContext(7));
+    lat = lat.setScale(7, RoundingMode.HALF_UP).stripTrailingZeros();
```

Comment:
@cammace Doesn't this need a call to `asDouble()` to make sure we write a `double` value to the `JsonArray`? Does it change the behavior in any way?